packages/nimble-components/src/label-provider/core/tests/label-provider-core.stories.ts

```diff
@@ -0,0 +1,22 @@
+import type { StoryObj } from '@storybook/html';
+import {
+    LabelProviderArgs,
+    labelProviderMetadata
+} from '../../base/label-provider-stories-utils';
+import { labelProviderCoreTag } from '..';
+import * as labelTokensNamespace from '../label-tokens';
+
+const metadata = {
+    ...labelProviderMetadata,
+    title: 'Tokens/Label Providers',
+    tags: ['autodocs']
```

Comment:
Could you add an MDX file for the label providers and disable autodocs? That would let us customize a few things:
- include stories for both providers on the docs page instead of just the core one
- consider hiding the "show code" button on the story. One way to do this is by importing it as `<Canvas>` rather than `<Story>` in the MDX
- add more markdown text providing context for each story

You can look at the button MDX for inspiration. I also have a local branch I can push if that would be helpful.

Reply:
Took a stab at that, let me know what you think.
- Hid "Show Code" - along with `Canvas`, that also required [sourceState="none"](https://storybook.js.org/docs/react/api/doc-block-canvas#sourcestate)
- Added section headings for each provider/story. Wasn't too sure what additional markdown/text to put in each though so the blurb is pretty short right now.